frontend/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="./index.js"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <title>Express Project</title>
+</head>
+<body>
+    <form id="addClass">
+        <input type="text" placeholder="Class Name"  id="classNameInput">
+        <input type="text" placeholder="Teacher" id="teacherNameInput">
+        <button>Add Class</button>
+    </form>
+    <br>
+    <form id="addStudent">
+        <input type="text" value="Class">
```

Comment:
This is not correct. You can't hardcode value in your inputs. You are making the user erase the typed in words and write their own. Not a good UX. You should have placeholders instead.